woocommerce/woocommerce
# Fix broken code coverage

### All Submissions:






### Changes proposed in this Pull Request:



1. Increased PHP Code Coverage workflow timeout to 20 mins.
2. Excluded `WC_Admin_Tests_Reports_Orders_Stats` test from the code coverage -- this test takes too long (6k lines of code 😮 ). We should refactor the test.
3. Added clean-up code to avoid changing global state.

### How to test the changes in this Pull Request:

1. Make sure `Run code coverage on PR / Code coverage (PHP 7.4, WP Latest) (pull_request) ` passes in this PR.
2. Optionally, you can run `pnpm nx test-code-coverage woocommerce` in your local.




### Changelog entry
N/A

### FOR PR REVIEWER ONLY:
